number1 = 5
number2 = 10
try:
    number1 = int(input("5"))
   
except ValueError:
    
    pass 
else
    print("No value error detected")
finally:
    print("Values taken care of")

try:5/10
  
    pass 
except ZeroDivisionError: incorrect
  pass  